hs_core/models.py

```diff
@@ -1576,7 +1576,7 @@ def rdf_triples(self, subject, graph):
         graph.add((coverage, RDF.value, Literal(value_string)))
 
     @classmethod
-    def validate_coverage_type_value_attributes(cls, coverage_type, value_dict):
+    def validate_coverage_type_value_attributes(cls, coverage_type, value_dict, use_limit_postfix=True):
```

Comment:
What are the changes in this file for and how are they related tot he model aggregation?  It looks like it was checked in by mistake.

Reply:
The changes in `validate_coverage_type_value_attributes` are to allow spatial coverage bounding box key names either as 'north', 'east', 'south', and 'west' or 'northlimit', 'eastlimit', 'southlimit', and 'westlimit'. Before this change, the function was expecting the key names to be 'northlimit', 'eastlimit', 'southlimit', and 'westlimit' if the coverage type is box and 'north' and 'east' if the coverage type is point. The serializer (https://github.com/hydroshare/hydroshare/blob/4626-rest-api-for-model-aggregation/hs_file_types/serializers.py#L21) that I am using for spatial coverage uses bounding box names as 'north', 'east', 'south', and 'west' for coverage type box and 'east' and 'north' for type point. In order for the serializer to use the function  `validate_coverage_type_value_attributes` I had to make these changes. 

This change should not affect hsmodels as the function `validate_coverage_type_value_attributes` is not used by hsmodels.